mail/api.py

```diff
@@ -136,7 +145,8 @@ def send_batch(cls, subject, body, recipients,  # pylint: disable=too-many-argum
             params = {
                 'to': emails,
                 'subject': subject,
-                'text': body,
+                'html': body,
+                'text': soup.get_text().strip(),
```

Comment:
Might also be worth moving generation of the `text` value outside the loop as well so we only incur that cost once too. I think we parameterize the templates using Mailgun vars so that should be ok.